### ISSUE TITLE
Assignment inside of latte is now handled like error in Latte 2.9.2

### ISSUE THREAD

**Issue by tprochazka** [uestla/TwiGrid#40]
The latest Latte fail with this library on 

```
The assignment in the {include #body-row,
					$recordVariable => $record,
					primaryString => ($primaryString = ...} looks like an error

```
Because of this
https://github.com/uestla/TwiGrid/blob/97f419365497bbab391d7a4ba5eed7c1ec14351a/src/TwiGrid/DataGrid.latte#L355

Latte changelog:
https://github.com/nette/latte/releases/tag/v2.9.2

**Comment by uestla** [comment]
@tprochazka This should be fixed in current Latte (see https://github.com/nette/latte/issues/258), can you please test it?

**Comment by tprochazka** [comment]
I just tested it with latte 2.10.1 and it is even worst

![image](https://user-images.githubusercontent.com/172094/107128538-bda95c00-68be-11eb-8beb-82fb741faf27.png)

Also line number is now wrong

**Comment by tprochazka** [comment]
It is fixed in latte/latte 2.10.2

**Comment by uestla** [comment]
Thanks!
